sdk/tests/tutorials/instruments/test_complex_instruments.py

```diff
@@ -3,7 +3,7 @@
 import lusid
 import lusid.models as models
 from datetime import datetime
-from utilities import TestDataUtilities
+#from utilities import TestDataUtilities
```

Comment:
If this import is no longer needed then the line should be deleted, although it looks like it is required (line 12) so best to remove the `#`.

Reply:
Initially did this to debug, have now removed the # so it is back in the newer version.